corehq/couchapps/deleted_users/views/by_username/map.js

```diff
@@ -0,0 +1,5 @@
+function(doc) {
```

Comment:
i think the convention we've been using in all the couchapps to ensure that people understand that there is only supposed to be one view per design doc is by specifying `views/view` in the path. so this would become

```
corehq/couchapps/deleted_users/views/view/map.js
```
or
```
corehq/couchapps/deleted_users_by_username/views/view/map.js
```